.gitignore

```diff
@@ -0,0 +1,7 @@
+# Compiled python
+*.pyc
+
+# CMake directory
+cmake_build/*
+!cmake_build/example_configure.bat.do_not_touch
```

Comment:
We should not.  Notice the ```!```
Basically I want to ignore everything there except those files